components/work_templates/index.tsx

```diff
@@ -292,6 +292,7 @@ const WorkTemplateModal = () => {
                     visibility={selectedVisibility}
                     onNameChanged={handleOnNameChanged}
                     onVisibilityChanged={handleOnVisibilityChanged}
+                    template={selectedTemplate}
```

Comment:
not a strong opinion on this but maybe instead of selectedTemplate we could send templateHasPlaybooks.

Reply:
I actually did consider doing this, but ended up feeling like we would have to calculate the content of the work template to get this info every time, even when we don't need it